docs/modeling-your-data/modeling-your-data-with-dbt/dbt-advanced-usage/dbt-incremental-logic/index.md

```diff
@@ -9,7 +9,7 @@ import TabItem from '@theme/TabItem';
 ```
 
 :::tip
-On this page, `<package>` can be one of: `web`, `mobile`, `ecommerce`, `normalize`
+On this page, `<package>` can be one of: `web`, `mobile`, `e-commerce`, `normalize`
```

Comment:
Nope, this one should have been without the hyphen.

Reply:
Ah yep I thought it was referring to the tags